### ISSUE TITLE
Cyrillic symbols not support in paths and filenames

### ISSUE THREAD

**Issue by Skyrimus** [aarcangeli/Serious-Sam-Android#32]
![Screenshot_20191006-195654](https://user-images.githubusercontent.com/5832920/66272534-a9d98a00-e873-11e9-8911-6c557ec76d84.png)
[SeriousSam.log](https://github.com/aarcangeli/Serious-Sam-Android/files/3694657/SeriousSam.log)

**Comment by Skyrimus** [comment]
Trouble with symbols encoding, I tried change it to utf-8, but it not works

**Comment by aarcangeli** [comment]
The string "Текстуры" where does it come from?
I suggest not using non-Latin file names, is it required for something?

**Comment by Skyrimus** [comment]
This string come from Russian modification, in this modification used Russian language to Textures, on Windows version this mod works fine, I think it will need fix

**Comment by Skyrimus** [comment]
In path to some textures used cyrillic symbols as Russian language

**Comment by Skyrimus** [comment]
Do you have some ideas to fix it? If I change homeDir path SeriousSam https://github.com/Skyrimus/Serious-Sam-Android/blob/master/app/src/main/java/com/github/aarcangeli/serioussamandroid/MainActivity.java#L498 to some cyrillic symbols  it works fine

**Comment by aarcangeli** [comment]
Where is stored Tp.tex? in a gro file?

**Comment by Skyrimus** [comment]
Change SeriousSam path to Сэм path works, but path to Текстуры not works 
![Screenshot_20191009-004157](https://user-images.githubusercontent.com/5832920/66435584-dcc77d80-ea2d-11e9-8c02-8c5e112acb30.png)
![Screenshot_20191009-004342](https://user-images.githubusercontent.com/5832920/66435585-dcc77d80-ea2d-11e9-9a82-1930ebfbba47.png)

**Comment by Skyrimus** [comment]
@aarcangeli yeah, in gro, and i tried to unzip Текстуры to game path and it not works too

**Comment by Skyrimus** [comment]
This some more ?????? is Текстуры

**Comment by Skyrimus** [comment]
How to change path to Текстуры in utf encoding?

**Comment by Skyrimus** [comment]
How can I contact you? Facebook?

**Comment by aarcangeli** [comment]
How about telegram? my username is http://t.me/alessandroarcangeli

**Comment by Skyrimus** [comment]
@aarcangeli ok.

**Comment by Skyrimus** [comment]
@aarcangeli i can t write you, please add me 
https://t.me/Skyrimus

**Comment by aarcangeli** [comment]
This is now fixed on master branch
